lib/usd/translators/shading/usdStandardSurfaceWriter.cpp

```diff
@@ -140,13 +140,12 @@ void PxrUsdTranslators_StandardSurfaceWriter::Write(const UsdTimeCode& usdTime)
                 PxrMayaUsdPreviewSurfaceTokens->UseSpecularWorkflowAttrName, SdfValueTypeNames->Int)
             .Set(1, usdTime);
 
-        AuthorShaderInputFromScaledShadingNodeAttr(
+        AuthorShaderInputFromShadingNodeAttr(
```

Comment:
The specular color is not scaled by the specular coefficient in the PBR implementation, so we should not scale here.